std-lib/bool/bool.lurk

```diff
@@ -0,0 +1,20 @@
+;; some boolean operations 
+
+;; determine if two bools are equal 
+(let ((eq_bool (lambda (x y)
+    (if x
+        (if y
+            t
+            nil)
+        (if y
+            nil
+            t )))))
+    (current-env))
```

Comment:
If the goal is just to preserve truth value, then I think this can just be:
```
(if x  y (eq y nil))
```

Reply:
this particular function loops infinitely if you give it `nil`, but using this style I made the `bool` code much shorter and sleeker

Reply:
good call, implemented